src/PerfView/PerfViewData.cs

```diff
@@ -8359,6 +8367,7 @@ protected override Action<Action> OpenImpl(Window parentWindow, StatusBar worker
             bool hasGC = false;
             bool hasJIT = false;
             bool hasAnyStacks = false;
+            bool hasDotNetHeapDumps = true;
```

Comment:
Erm, apologies if I'm missing the point completely, but shouldn't this default to `hasDotNetHeapDumps = false`, then set it to `true` only if the event starts with `GC/BulkNode`? 

Other wise it'll always try and add a `PerfViewHeapSnapshots`?